src/Logging/Logging/src/LoggerFactory.cs

```diff
@@ -69,6 +91,11 @@ private void RefreshFilters(LoggerFilterOptions filterOptions)
             }
         }
 
+        /// <summary>
+        /// Creates an <see cref="ILogger"/> with the given <paramref name="categoryName"/>.
+        /// </summary>
+        /// <param name="categoryName">The categoryName for messages produced by the logger.</param>
```

Comment:
In the prose, use `category name` (it's just English here, not a code identifier).